packages/projects-docs/pages/learn/repositories/themes.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Themes
+description: Change the color of your editor by utilizing the preset themes available in VS Code.
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+# Themes
+
+Change the color of your editor by utilizing the preset themes available in VS Code.
+
+![CodeSandbox Themes](../images/themes.jpg)
+
+## Overview
+
+CodeSandbox maps themes of Visual Studio Code extensions to apply over the editor. Currently, it only offers a set of preset themes, including various dark and light themes:
```

Comment:
Same as above

```CodeSandbox maps popular themes from the dev community to apply over the editor.```

Reply:
Proposing an updated version below that might be easier to read:

"CodeSandbox provides some popular themes created by the dev community that you can apply to the cloud editor. Currently, we only offer a few preset themes, with dark and light options:"